Dockerfile

```diff
@@ -21,7 +21,7 @@ WORKDIR /var/www/html
 
 ###
 
-FROM nginx:1.26-alpine as nginx-module-builder
+FROM nginx:1.26.1-alpine AS nginx-module-builder
```

Comment:
This hass been pinned to address version drift between `nginx` and `nginxinc/nginx-unprivileged`